### PR TITLE
michaelcontribution

### PR DESCRIPTION
**Pull request by Michaelin007**
I added my own card using the template provided

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -115,6 +115,30 @@ <h2>A project for first-time contributions</h2>
...
+              </li>
+              <li>
+                <a href="https://dev.to/michaelin007/multiple-page-web-scrapper-in-rust-3hnl" target="_blank" title="Second Resource">Multiple Page Web Scraper in Rust</a>
+              </li>
+              <li>
+                <a href="https://github.com/" target="_blank" title="Third resource">Github</a>
+              </li>
+            </ul>
+          </div>
+        </div>
```
Comment (TenTraicion) [inline index.html:140]:
```suggestion
        </div>
        <!-- ________michaelin007 Contributor card END ________ -->
```

**Review by TenTraicion — CHANGES_REQUESTED** [review]
@Michaelin007, This card is okay but missing the ending comment. Kindly do the requested change and commit! Thanks!

File: index.html (hunk 1 above)
Comment (Michaelin007) [reply index.html:140]: ending comment added

**Review by nzaih1999 — APPROVED** [review]
Hello @Michaelin007 I've approved your changes..your card will be live in a few minutes..happy coding!

**Review by TenTraicion — APPROVED** [review]

**Comment by TenTraicion** [comment]
@Michaelin007, Thanks for your Contribution. and Congrats on your first PR Merge in this project. Your Card will be live within ~2-3 minutes, which can be found: [Here](https://syknapse.github.io/Contribute-To-This-Project/)!

**Pull request merged**